### PR TITLE
MBS-8996: Display merge append errors correctly

### PR DESCRIPTION
**Pull request by mwiencek**
`[% form.errors %]` can be an array if there are multiple errors. The only errors that should be possible to trigger by direct user input are for the medium positions, so just display the errors for those fields inline.

### CODE REVIEW

File: root/release/merge.tt [hunk 1]
```diff
@@ -63,12 +61,18 @@
...
+                th span.error { margin: 0 12px 0 6px; }
+              </style>
               [% FOR medium=mediums %]
               [% field = form.field('medium_positions').field('map').field(loop.index) %]
               <tr class="subh">
                 <th colspan="4">
                   <label>[% l('New position:') %]</label>
                   [% r.text(field.field('position'), size=2) %]
+                  [%~ IF field.field('position').has_errors;
+                        '<span class="error">' _ html_escape(field.field('position').errors.0) _ '</span>';
```
Comment (gentlecat) [inline root/release/merge.tt]: `.errors.0` is a description?

Comment (mwiencek) [inline root/release/merge.tt]: Yeah, `errors` is an array of error strings for the field: http://search.cpan.org/dist/HTML-FormHandler-0.40058/lib/HTML/FormHandler/Manual/Errors.pod

In TT syntax `.errors.0` means `->errors->[0]`, so this only displays the first error if there are multiple. But that should only happen if they enter a 0 or negative position multiple times...

Comment (mwiencek) [inline root/release/merge.tt]: Here's where those error strings come from: https://github.com/metabrainz/musicbrainz-server/blob/2835d40ce94514d1569e50e2abf3ec6764784c40/lib/MusicBrainz/Server/Form/Merge/Release.pm#L62-L62

**Comment by gentlecat** [comment]
:shipit:

**Pull request merged**